npm/npm_import.bzl

```diff
@@ -355,6 +355,13 @@ WARNING: `package_json` attribute in `npm_translate_lock(name = "{name}")` is de
     if type(bins) != "dict":
         fail("Expected bins to be a dict")
     for key, value in bins.items():
+        if type(value) == "list":
+            # The passed 'bins' value is already in the dict-of-string-list
+            # form needed by the rule. This is undocumented but necessary for
+            # the bzlmod interface to use this macro since dict-of-dicts attributes
+            # cannot be passed into module extension attrs.
+            bins_string_list_dict = bins
+            break
```

Comment:
I hope stardoc will soon generate docs for module extensions.

Reply:
Should I document this in the macro docstring or leave undocumented for now?

Reply:
Lets just change the WORKSPACE version to be the same format in a follow-up and change the docs there. The current syntax can be supported as an undocumented variant using typeof.